### DISCUSSION TITLE
115报错ailed link: failed get link: post "https://proapi.115.com/app/chrome/downurl?t=1728646849": net/http: invalid header field value for "cookie"

### DISCUSSION THREAD

**Discussion by wcyattz** [AlistGo/alist#7317] [Q&A]
115报错，提示ailed link: failed get link: post "https://proapi.115.com/app/chrome/downurl?t=1728646849": net/http: invalid header field value for "cookie"，怎么解决

**Comment by hehxuan** [comment]
我的也是同样问题

**Comment by optimuszhao** [comment]
我也遇到一样的问题，希望今早解决，下载alist不可以用了，很糟糕啊

**Comment by JiaqiFang** [comment]
我的也是同样问题
